muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt

```diff
@@ -67,26 +70,34 @@ var MTableCellProps.variant by EnumPropToString(MTableCellVariant.values())
 
 fun RBuilder.mTableCell(
         key: Any? = null,
-        variant: MTableCellVariant = MTableCellVariant.body,
-        sortDirection: MTableCellSortDirection = MTableCellSortDirection.False,
-        align: MTableCellAlign = MTableCellAlign.inherit,
-        padding: MTableCellPadding = MTableCellPadding.default,
+        variant: MTableCellVariant? = null,
+        sortDirection: MTableCellSortDirection? = null,
+        align: MTableCellAlign? = null,
+        padding: MTableCellPadding? = null,
         size: MTableCellSize? = null,
         colSpan: Int? = null,
         component: String? = null,
         scope: String? = null,
-
         className: String? = null,
-        handler: StyledHandler<MTableCellProps>? = null) = createStyled(TableCellComponent) {
-    attrs.align = align
+        handler: StyledHandler<MTableCellProps>? = null
+) = createStyled(TableCellComponent) {
+    if (align != null) {
```

Comment:
For these if null params, probably should keep with the rest of the style and be
`align?.let { attrs.align = it }`

Reply:
em..Should keep same style, I just used quick fix with idea